scss/civicrm/contact/pages/_export-contact.scss

```diff
@@ -1,9 +1,8 @@
 .CRM_Export_Form_Select {
```

Comment:
this selector doesn't seem related specifically to "export contact", it has a pretty generic name. Which is it then? is it only for export contact or is it for any "export XYZ" page? if it's the latter, then any style contained in it shouldn't be in a partial called `_export-contact.scss`

Reply:
I have renamed the file to `_export.scss` because all styling in it is for all export pages, not just for contacts.